lib/mi_bridges/driver/medical_bills_page.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module MiBridges
+  class Driver
+    class MedicalBillsPage < BasePage
+      TITLE = "Medical Bills"
+
+      delegate(
+        :medical_expenses,
+        :primary_member,
```

Comment:
we are filling in all of this info for primary member rather than for specific members. Let's keep track of that via this spreadsheet: https://docs.google.com/spreadsheets/d/1je97jHCu98-yFM5xYerOCb8pf1rPIp6k8QQE3PVc_lE/edit#gid=0

(Alan asked me to record places where MI Bridges and our app are inconsistent for future review)

Reply:
@jessieay for michigan benefits we collect this info only for the primary member (it's on the SnapApplications model).  In MI Bridges it only asks for that information for the primary member. It doesn't present those questions per member.

Reply:
just gave you access - sorry for the delay!

Are you sure it isn't only showing this questions for the primary member because we indicate that the primary member has medical expenses?